Dockerfile

```diff
@@ -0,0 +1,63 @@
+ARG GO_VERSION=1.23.4
```

Comment:
The file looks like the same as used in sonar-iac - maybe it should be moved to build-logic and reused?

Reply:
Good suggestion, however looks like Cirrus cannot build a Docker image from a file in submodules. It would need to clone the submodule first.